### PR TITLE
added bookkeeping to the results dictionary

### PR DESCRIPTION
**Pull request by profjsb**
added two new key/values based on the original L-S refine script.

### CODE REVIEW

File: cesium/features/lomb_scargle.py [hunk 1]
```diff
@@ -25,6 +26,13 @@ def lomb_scargle_model(time, signal, error, sys_err=0.05, nharm=8, nfreq=3, tone
     nfreq : int
         Number of frequencies to fit.
 
+    freq_grid : str
```
Comment (stefanv) [inline cesium/features/lomb_scargle.py]: Looks like this can be one argument: freq_grid. If specified, it is used, otherwise determined automatically.

Comment (profjsb) [reply cesium/features/lomb_scargle.py]: done

File: cesium/features/lomb_scargle.py [hunk 2]
```diff
@@ -51,6 +52,14 @@ def lomb_scargle_model(
         Order of polynomial to fit to the data while
         fitting the dominant frequency. Defaults to 1.
 
+    freq_grid : dict or None, optional
+        Grid parameters to use. If None, then calculate the frequency
+        grid automatically. To supply the grid, keys
+        ["f0", "fmax"] are expected. "f0" is the smallest frequency
```
Comment (stefanv) [inline cesium/features/lomb_scargle.py:58]: Where do `df` and `numf` come from?

Comment (profjsb) [reply cesium/features/lomb_scargle.py:58]: they are either supplied by the user or calculated using lines 87 & 89.

**Pull request merged**